kubernetes/beta/ambassador/ambassador-rbac-beta-public.yaml

```diff
@@ -36,7 +36,13 @@ rules:
   verbs: ["get", "list", "watch"]
 - apiGroups: [ "networking.internal.knative.dev" ]    
   resources: [ "ingresses/status", "clusteringresses/status" ]    
-  verbs: ["update"]  
+  verbs: ["update"]
+- apiGroups: [ "extensions" ]
+  resources: [ "ingresses" ]
+  verbs: ["get", "list", "watch"]
+- apiGroups: [ "extensions" ]
```

Comment:
`apiGroups: [ "extensions" ]` seem to be duplicate

Reply:
This is upstream code 
https://getambassador.io/yaml/ambassador/ambassador-rbac.yaml